flexget/plugins/metainfo/nfo_lookup.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+import os
+import xml.etree.ElementTree as ET
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugins.metainfo.imdb_lookup import ImdbLookup
+# from flexget.utils.imdb import make_url as make_imdb_url
+# from flexget.utils.imdb import ImdbSearch
+
+log = logging.getLogger('nfo_lookup')
+
+
+class NfoLookup(object):
+    """
+    Retrieves information from a local '.nfo' info file.
+
+    The nfo file is used by Kodi.
+
+    Example:
+        nfo_lookup: yes
+
+    WARNING: This plugin will read a file with extension '.nfo' and the same
+    name as the entry filename as an XML file using xml.etree.ElementTree from
+    the standard python library. As such, it is vulnerable to XML
+    vulnerabilities described in the link below
+    https://docs.python.org/3/library/xml.html#xml-vulnerabilities
+
+    Use this only with nfo files you have created yourself.
+    """
+    # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+    # Fields that will be read from the nfo file. The corresponding field in
+    # the entry will be nfo_{0} for each field name in these two lists.
+
+    # Fields that appear only once in the nfo file
+    single_value_fields = ["title", "originaltitle", "sorttitle", "rating",
+                           "year", "votes", "plot", "runtime",
+                           "id", "filenameandpath", "trailer",
+                           # thumb has multiple values, but we will get the first one
+                           "thumb"]
+    # Fields that can appear multiple times in the nfo file
+    multiple_value_fields = [
+        # "thumb",
```

Comment:
Originally I was going to get other fields such as actors and directors as well from the nfo file. However, the nfo file only has the names and I would have to get the IMDB id of each of them to add to the fields `imdb_actors` and `imdb_directors`. 

For fields such as title and plot the nfo file might have different information, such as the translated title and plot. On the other hand, the names of actors and directors will be the same and thus I commented out this part to let the IMDB search fill the data.

If IMDB search is not performed anymore as part of this plugin then probably this should be kept and improved.